### PR TITLE
chore(gitignore): Add husky content only into gitignore

### PR DESCRIPTION
**Pull request by fgchaio**
I was having issues running the local environment because the .husky folder is added into the .gitignore file.
With these changes, only the content will be ignored.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,5 +24,4 @@ yarn-error.log*
 
 /.vscode
 /.eslintcache
-.idea
-/.husky/
+.idea
```
Comment (cbernat) [inline .gitignore:27]: Why not this?
```suggestion
.idea
/.husky/**/**.*
```

Comment (fgchaio) [reply .gitignore:27]: Because if I do it like that git wants to add the folder content into de project.

Comment (cbernat) [reply .gitignore:27]: But with your changes, I remove the .husky folder, then do a `yarn install` and when I try to commit, husky files and folder appear. 
image

**Comment by fgchaio** [comment]
I'm closing this PR. This doesn't seem to be an issue to others. I will update yarn to the latest version and it might solve it.

**Pull request closed without merging**